src/main/java/edu/hawaii/its/api/controller/OotbRestController.java

```diff
@@ -30,26 +29,30 @@
 @RequestMapping("/api/groupings/ootb")
 public class OotbRestController {
     private static final Log logger = LogFactory.getLog(OotbRestController.class);
-    private final PolicyFactory policy;
 
     @Value("${url.api.2.1.base}")
     private String API_2_1_BASE;
 
-    @Autowired
-    private UserContextService userContextService;
+    private final PolicyFactory policy;
+
+    private final UserContextService userContextService;
 
-    @Autowired
-    private OotbStaticUserAuthenticationFilter ootbAuthenticationFilter;
+    private final OotbStaticUserAuthenticationFilter ootbAuthenticationFilter;
 
-    @Autowired
-    private OotbActiveUserProfileService ootbActiveUserProfileService;
+    private final OotbActiveUserProfileService ootbActiveUserProfileService;
 
-    @Autowired
-    private HttpRequestService httpRequestService;
+    private final HttpRequestService httpRequestService;
 
     // Constructor.
-    public OotbRestController() {
+    public OotbRestController(UserContextService userContextService,
+            OotbStaticUserAuthenticationFilter ootbAuthenticationFilter,
+            OotbActiveUserProfileService ootbActiveUserProfileService,
+            HttpRequestService httpRequestService) {
         policy = Sanitizers.FORMATTING;
```

Comment:
Unrelated to your PR, but I wonder if we can create this policy variable when we declare it above?

Reply:
I think the policy can be declared above

Reply:
However, I see most of the policies are declare inside the constructor, please let me know if we want to change it

Reply:
I think we should declare the `policy` above so that the constructor's main purpose is to inject services.

Reply:
I will move the policy above